src/Compilers/Core/Portable/CommandLine/CommonCompiler.cs

```diff
@@ -1169,6 +1169,7 @@ private int RunCore(TextWriter consoleOutput, ErrorLogger errorLogger, Cancellat
                             emitTestCoverageData: emitOptions.EmitTestCoverageData,
                             pePdbFilePath: emitOptions.PdbFilePath,
                             privateKeyOpt: privateKeyOpt,
+                            codePage: emitOptions.DefaultSourceFileEncoding,
```

Comment:
>codePage [](start = 28, length = 8)

Nit: call this parameter `defaultSourceFileEncoding`. "codepage" does not say much more than "encoding", but not the purpose.